com.unity.render-pipelines.universal/Editor/Converter/Converters/RenderSettingsConverter.cs

```diff
@@ -0,0 +1,301 @@
+
+            // Additional Lights
+            asset.additionalLightsRenderingMode = settings.PixelLightCount <= 1
+                ? LightRenderingMode.Disabled
+                : LightRenderingMode.PerPixel;
+            asset.maxAdditionalLightsCount = Mathf.Max(0, settings.PixelLightCount - 1);
+            asset.supportsAdditionalLightShadows = settings.Shadows != ShadowQuality.Disable;
+            asset.additionalLightsShadowmapResolution =
+                GetEquivalentAdditionalLightAtlasShadowResolution((int) settings.ShadowResolution);
+
+            // Reflection Probes
+            asset.reflectionProbeBlending = m_GraphicsTierSettings.ReflectionProbeBlending;
+            asset.reflectionProbeBoxProjection = m_GraphicsTierSettings.ReflectionProbeBoxProjection;
+
+            // Shadows
+            asset.shadowDistance = settings.ShadowDistance;
+            asset.shadowCascadeCount = m_GraphicsTierSettings.CascadeShadows ? settings.ShadowCascadeCount : 0;
+            asset.cascade2Split = settings.CascadeSplit2;
+            asset.cascade4Split = settings.CascadeSplit4;
+            asset.supportsSoftShadows = settings.Shadows == ShadowQuality.All;
+        }
+
+        private static int GetEquivalentMainlightShadowResolution(int value)
+        {
+            return GetEquivalentShadowResolution(value);
+        }
+
+        private static int GetEquivalentAdditionalLightAtlasShadowResolution(int value)
+        {
+            return GetEquivalentShadowResolution(value);
+        }
+
+        private static int GetEquivalentShadowResolution(int value)
+        {
+            switch (value)
+            {
+                case 0: // low
+                    return 1024;
+                case 1: // med
+                    return 2048;
+                case 2: // high
+                    return 4096;
+                case 3: // very high
+                    return 4096;
+                default: // backup
+                    return 1024;
+            }
+        }
+
+        private RenderingMode GetEquivalentRenderMode(RenderingPath path)
+        {
+            switch (path)
+            {
+                case RenderingPath.VertexLit:
+                case RenderingPath.Forward:
+                    return RenderingMode.Forward;
+                case RenderingPath.DeferredLighting:
+                case RenderingPath.DeferredShading:
+                    return RenderingMode.Deferred;
+                default:
+                    return RenderingMode.Forward;
+            }
+        }
+
+        #region Structs
+
+        private struct GraphicsTierSettings
+        {
+            public bool ReflectionProbeBoxProjection;
+            public bool ReflectionProbeBlending;
+            public bool CascadeShadows;
+            public bool HDR;
+            public RenderingPath RenderingPath;
+        }
+
+        private class SettingsItem { }
+
+        private class RenderSettingItem : SettingsItem
```

Comment:
Do we need both classes?

Reply:
Again this is part fo future proofing, I actually had the camera class here too, can remove now if you think it's worth it otherwise it will be used in the following camera PR.

Reply:
I think its okay to keep 👍